src/content/1.7/modules/sell/techvalidation-keysteps.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Technical validation - Key steps
+---
+
+# Technical Validation : Key steps
+
+Have you ever wondered how the PrestaShop Addons Team treats the validation ?
+
+We are going to show you how to win time so you can pretty much "validate" your own module before submitting it on the Marketplace Addons.
+First you might want to read the [Technical Tools]({{<ref "1.7/modules/sell/technical-tools" >}}) and the [Good Practices]({{<ref "1.7/modules/creation/good-practices" >}}). 
+
+### 1. The Basics
+
+As most of you know, there are 3 types of submittions : New, Update min, Update maj.
+We do not handle them with the same standards:
+
+- We are very strict and meticulous when it comes to New modules.
+- For major updates, we have the exact same verifications than the New modules.
+- For minor updates, we verify the validator and we also have a Diff Tool to only see the modifications.
+
+Let's now give you more details about how we handle shit.
```

Comment:
🤣I dont think "shit" will be accepted by the master of wordings @LouiseBonnard

Reply:
Yeah i'm very sorry. I was going too fast to create an introduction so I could focus on the main content. I intended to rewrite that but totally forgot.

The danger of not having time to start and end it in a single day :(